LOAN/Loans/LoanBasicTerms.rdf

```diff
@@ -597,199 +226,33 @@
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;loanInterestPaymentFrequency">
-		<rdfs:label xml:lang="en">loan interest payment frequency</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-loan-loan;LoanInterestPaymentTermsSet"/>
-		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;RecurrenceInterval"/>
-		<skos:definition xml:lang="en">Payment Frequency - Frequency of payments due, i.e. number of months between payments.</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;loanInterestRateCap">
-		<rdfs:label xml:lang="en">loan interest rate cap</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-loan-loan;LoanInterestPaymentTermsSet"/>
-		<rdfs:range rdf:resource="&fibo-fnd-utl-val;PercentageValue"/>
-		<skos:definition xml:lang="en">The cap on the interest rate chargeable, if there is one.</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:DatatypeProperty rdf:about="&fibo-loan-loan-loan;loanInterestRateType">
-		<rdfs:label xml:lang="en">loan interest rate type</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-loan-loan;LoanInterestPaymentTermsSet"/>
-		<skos:definition xml:lang="en">Loan Interest Rate Type.</skos:definition>
-	</owl:DatatypeProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;loanInterestResetInterval">
-		<rdfs:label xml:lang="en">loan interest reset interval</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-loan-loan;LoanInterestPaymentTermsSet"/>
-		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Duration"/>
-		<skos:definition xml:lang="en">Interest Rate Reset interval - The interval in months at which the interest rate is adjusted (for floating loans)</skos:definition>
-	</owl:ObjectProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;loanMaximumAllowedBalance">
 		<rdfs:label xml:lang="en">loan maximum allowed balance</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+		<rdfs:domain rdf:resource="&fibo-loan-ln-ln;Loan"/>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 		<skos:definition xml:lang="en">Maximum balance - For loans with flexible re-draw facilities, the maximum loan amount that could potentially be outstanding</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-loan-loan-loan;loanOriginalAdvanceAmount">
 		<rdfs:label xml:lang="en">loan original advance amount</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-loan-ln-ln;LoanContract"/>
+		<rdfs:domain rdf:resource="&fibo-loan-ln-ln;Loan"/>
```

Comment:
How does this relate to the Principal?

Reply:
The rest of this ontology needs lots of work - I stripped a lot of things out of it and merged them into LoanCore if they were not already covered, but have not worked through the rest of it.